File: HackerRank/birthdayCandles/candles.rb
#!/bin/ruby

require 'json'
require 'stringio'

# Complete the birthdayCakeCandles function below.
def birthdayCakeCandles(ar)
    tallest = 0
    ar.each { |candle| tallest = candle if candle > tallest}
    return ar.count { |candle| candle === tallest }
end

fptr = File.open(ENV['OUTPUT_PATH'], 'w')

ar_count = gets.to_i

ar = gets.rstrip.split(' ').map(&:to_i)

result = birthdayCakeCandles ar

fptr.write result
fptr.write "\n"

fptr.close()

File: HackerRank/birthdayCandles/candles.py
#!/bin/python3

import math
import os
import random
import re
import sys

# Complete the birthdayCakeCandles function below.
# def birthdayCakeCandles(ar):
#     tallest = 0
#     for candle in ar:
#         if candle > tallest:
#             tallest = candle
#     return ar.count(tallest)

def birthdayCakeCandles(ar):
    return sorted(ar)[-1]

if __name__ == '__main__':
    fptr = open(os.environ['OUTPUT_PATH'], 'w')

    ar_count = int(input())

    ar = list(map(int, input().rstrip().split()))

    result = birthdayCakeCandles(ar)

    fptr.write(str(result) + '\n')

    fptr.close()
